### PR TITLE
Create and apply initial migration in the @edgedb/create CLI; minor CLI changes

### PR DESCRIPTION
**Pull request by beerose**
Note: updated README.md files will follow. 

Also added this outro message:
CleanShot 2024-02-07 at 13 28 25@2x

### CODE REVIEW

File: packages/create/src/recipes/_edgedb/index.ts [hunk 1]
```diff
@@ -117,6 +117,19 @@ const recipe: Recipe<EdgeDBOptions> = {
...
+      try {
+        await execInLoginShell("edgedb migration create", { cwd: projectDir });
+        await execInLoginShell("edgedb migrate", { cwd: projectDir });
+        spinner.stop("Initial migration created and applied");
+      } catch (error) {
+        logger(error);
+        throw error;
+      } finally {
+        spinner.stop();
+      }
```
[the remark this answers is not in the document]
Comment (beerose) [reply packages/create/src/recipes/_edgedb/index.ts:132]: Oh, and that also depends on whether someone picked "yes" for "Initialize a new EdgeDB project with edgedb project init?"

File: packages/create/package.json [hunk 2]
```diff
@@ -25,6 +27,7 @@
   },
   "dependencies": {
     "@clack/prompts": "^0.7.0",
+    "chalk": "^5.3.0",
```
[the remark this answers is not in the document]
Comment (scotttrinh) [reply packages/create/package.json:30]: I don't have a strong preference here, but I just peeked in `@clack/prompts` to [see what they use for coloring](https://github.com/natemoo-re/clack/blob/90f8e3d762e96fde614fdf8da0529866649fafe2/packages/prompts/package.json#L57) and they use https://github.com/alexeyraspopov/picocolors which seems to be pretty popular too so there is a chance we can reduce bundle size slightly by using that.

**Review by scotttrinh — APPROVED** [review]
No blockers here! You're welcome to do the `chalk` -> `picocolors` and/or moving the intro and outro to `cli.ts` at your preference. Awesome job. Adding the intro and outro are nice touches!

File: packages/create/src/recipes/_install/index.ts [hunk 3]
```diff
@@ -1,5 +1,6 @@
 import debug from "debug";
 import * as p from "@clack/prompts";
+import pc from "picocolors";
```
Comment (beerose) [inline packages/create/src/recipes/_install/index.ts:3]:
```suggestion
```

**Pull request merged**